lib/location/location_search_delegate.dart

```diff
@@ -479,6 +500,22 @@ class _SuggestionList extends StatelessWidget {
       case 'tourism:apartment':
         return Icons.local_hotel;
 
+      // Icons used for the new feature: my places
```

Comment:
remove "new", cause soon it will be an established feature :-)

Reply:
you're right.